### PR TITLE
Update description for /beacon/validators endpoint.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## PR Description
Update description for /beacon/validators endpoint.

### CODE REVIEW

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java [hunk 1]
```diff
@@ -67,33 +67,31 @@ public GetValidators(final ChainDataProvider chainDataProvider, final JsonProvid
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.GET,
-      summary = "Returns validators that match the specified query.",
+      summary = "Get validators that match the specified query.",
       tags = {TAG_BEACON},
       description =
-          "Returns validator information. If no parameters specified, the first page of validators as of the current epoch are returned.",
+          "Returns validator information. Returns the first page of validators in the current epoch if you do not specify any parameters.",
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java]: can i buy some line breaks? if you use intellij just pressing 'enter' inside the string will close the string, add a '+' and restart the string on the next line...

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java [hunk 2]
```diff
@@ -67,33 +67,31 @@ public GetValidators(final ChainDataProvider chainDataProvider, final JsonProvid
...
       queryParams = {
-        @OpenApiParam(
-            name = EPOCH,
-            description = "Epoch to query. If not specified, current epoch is used."),
+        @OpenApiParam(name = EPOCH, description = "Epoch to query. Defaults to the current epoch."),
         @OpenApiParam(
             name = ACTIVE,
             description =
-                "If specified, return only validators which are active in the specified epoch."),
+                "Only return validators that are active in the specified `epoch`. By default, returns inactive and active validators.\n\n**Note**: This field accepts any value when specifying active validators."),
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java]: same here, if we could break it down...

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java [hunk 3]
```diff
@@ -67,33 +67,34 @@ public GetValidators(final ChainDataProvider chainDataProvider, final JsonProvid
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.GET,
-      summary = "Returns validators that match the specified query.",
+      summary = "Get validators that match the specified query.",
       tags = {TAG_BEACON},
       description =
-          "Returns validator information. If no parameters specified, the first page of validators as of the current epoch are returned.",
+          "Returns validator information.\n\n"
```
Comment (rolfyone) [inline data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java:73]: the actual string line breaks were optional, but i'm guessing this is formatting you actually want, so all good :)

**Review by rolfyone — APPROVED** [review]

File: data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java (hunk 3 above)
Comment (bgravenorst) [reply data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java:73]: Yes, I wanted those breaks :)

**Pull request merged**